lib/ui/tools/debug/app_storage_viewer.dart

```diff
@@ -158,6 +158,8 @@ class _AppStorageViewerState extends State<AppStorageViewer> {
                             ),
                             menuItemColor:
                                 getEnteColorScheme(context).fillFaint,
+                            pressedColor:
+                                getEnteColorScheme(context).fillFaintPressed,
```

Comment:
Not a blocker for this PR, but would be nicer if MenuItem had this built in whenever it is acting as a button and can be interacted with. Just so that there is one place that can be changed, and we don't need to repeat this whenever we create such items.

---

### Details

Menu item can be used both as an action (e.g in this case the font is bold):

<img width="492" alt="Screenshot 2022-12-30 at 11 38 05 AM" src="https://user-images.githubusercontent.com/24503581/210039913-721851da-6789-41de-8c60-721f95020df1.png">

Or it can be an row of information (in which case the font is regular, and the menu item is not interactable)

<img width="492" alt="Screenshot 2022-12-30 at 11 38 59 AM" src="https://user-images.githubusercontent.com/24503581/210039964-e95c9986-99ef-46b6-949e-d7d738b4ba95.png">

There is also the case of a nested menu item where the font is not bold but the item is still interactable:

<img width="586" alt="Screenshot 2022-12-30 at 11 40 24 AM" src="https://user-images.githubusercontent.com/24503581/210040022-3f40125f-2a93-4ada-a3ba-6ec7b8789f9d.png">